new_samples/README.md

```diff
@@ -0,0 +1,32 @@
+# Cadence Samples
+These are some samples to demonstrate various capabilities of Cadence client and server.  You can learn more about cadence at:
+* Cadence: https://github.com/uber/cadence
+* Cadence Client: https://github.com/uber-go/cadence-client
+
+## Prerequisite
+0. Install Cadence CLI. See instruction [here](https://cadenceworkflow.io/docs/cli/).
+1. Run the Cadence server: https://github.com/uber/cadence/blob/master/README.md
+2. Register the `cadence-samples` domain: https://cadenceworkflow.io/docs/cli/#quick-start or https://github.com/uber/cadence/blob/master/tools/cli/README.md
+
+
+
+## Steps to run samples
+Simply start the go binary by running
+```bash
+go run main.go
+```
+The worker will listen to `localhost:3000`. Then you may interact with the samples using Cadence CLI.
```

Comment:
no longer listening to localhost (nor did it do anything useful when it did)

Reply:
yeah this was a miss sorry about that. Has updated